src/Tizen.Multimedia/AudioManager/AudioStreamPolicy.cs

```diff
@@ -320,12 +329,14 @@ public void Dispose()
         /// <since_tizen> 3 </since_tizen>
         protected virtual void Dispose(bool disposing)
         {
-            if (!_disposed)
+            if (_disposed)
             {
-                if (_handle != null)
-                {
-                    _handle.Dispose();
-                }
+                return;
+            }
+
+            if (_handle != null)
```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/dispose-pattern
Dispose() only called when disposing was true

Reply:
I missed the _handle inherit safeHandle. I'll remove the finalizer again which is not necessary in this class and make some modifications in Dispose(bool disposing) again.

Reply:
It is now replaced with https://github.com/Samsung/TizenFX/pull/553